woocommerce/woocommerce-ios
# [Mobile Payments] Log payment failures

Fixes #5215 

To test:
- Complete an order with cart total ending in .05
- A payment failure will result
- Ensure it appears in debugger, e.g.:

```
2021-10-13 15:14:00.583714-0700 WooCommerce[8439:6310483] Failed to collect payment: paymentCapture(underlyingError: Hardware.UnderlyingError.paymentDeclinedByPaymentProcessorAPI)
```

FWIW, it looks like this to the user:

![IMG_0192](https://user-images.githubusercontent.com/1595739/137220300-037762f7-a8fc-43e8-a279-63a1645874ea.PNG)

The bundler moved the screenshots line too without need to, but I included it anyways since it will likely happen for the next person to build anyways

Update release notes:
